package.json

```diff
@@ -1,68 +1,74 @@
 {
-  "name": "karmatic",
```

Comment:
`npm install` will rewrite this back to spaces every time. Is there a way to exempt it from Prettier? In other projects I use an `.editorconfig` to control spacing for json/me separately from JS:

https://github.com/preactjs/preact-www/blob/master/.editorconfig

Reply:
It actually doesn't do that anymore; we handle `package.json` special in prettier. I just tested by installing a package; only one line changed.